app/controllers/storage_items_controller.rb

```diff
@@ -0,0 +1,72 @@
+class StorageItemsController < ApplicationController
+  before_action :set_storage_item, only: %i[ show edit update destroy ]
+
+  # GET /storage_items or /storage_items.json
+  def index
+    @storage_items = StorageItem.all
+  end
+
+  # GET /storage_items/1 or /storage_items/1.json
+  def show
+    @storage_item = StorageItem.find(params[:id])
+    @place = @storage_item.place
+  end
+
+  # GET /storage_items/new
+  def new
+    @storage_item = StorageItem.new
+  end
+
+  # GET /storage_items/1/edit
+  def edit
+  end
+
+  # POST /storage_items or /storage_items.json
+  def create
+    @storage_item = StorageItem.new(storage_item_params)
+
```

Comment:
When the storage item is created it will probably need a place associated with it (and the place_id should be in the parameters, because the url will be something like `/places/123/storage_items`, where 123 is the place_id